xbmc/Application.cpp

```diff
@@ -1377,7 +1377,7 @@ void CApplication::UnloadSkin(bool forReload /* = false */)
   else if (!m_saveSkinOnUnloading)
     m_saveSkinOnUnloading = true;
 
-  CGUIComponent *gui = CServiceBroker::GetGUI();
+  IGUIComponent *gui = CServiceBroker::GetGUI().get();
```

Comment:
```suggestion
  auto gui = CServiceBroker::GetGUI();
```
this I think looks better

Reply:
definitely - changed ...